core/eth/client.go

```diff
@@ -101,6 +106,32 @@ func (client *CallerSubscriberClient) GetERC20Balance(address common.Address, co
 	return numLinkBigInt, nil
 }
 
+const aggregatorAnswerFunctionID = "7e1b4cb0"
```

Comment:
Is this a Chainlink specific address?

Reply:
Yes it is. It's a literal that's tied a method in one of our smart contracts. @se3000 can elaborate if you have more questions.

Reply:
Ok great, maybe can you mention this as a comment for the layman?